packages/devtools_app/lib/src/debugger/program_explorer_model.dart

```diff
@@ -161,6 +161,10 @@ class VMServiceObjectNode extends TreeNode<VMServiceObjectNode> {
     }
 
     for (final clazz in lib.classes) {
+      // Don't surface synthetic classes created by mixin applications.
+      if (clazz.name.contains('&')) {
```

Comment:
if `class A with B, C` exists, will the name be `A&B&C`, and in that case, wouldn't we still want class `A` to be surfaced, even though we don't want `B` or `C`?

Reply:
They actually show up as different classes. In your example, we would currently show the following class entries:

- `A`
- `A&B`
- `A&B&C`